akka-actor/build.sbt

```diff
@@ -5,5 +5,9 @@ Formatting.formatSettings
 OSGi.actor
 Dependencies.actor
 Version.versionSettings
+unmanagedSourceDirectories in Compile += {
+  val base = scalaVersion.value.take(4)
+  sourceDirectory.value / s"main-$base" / "scala"
+}
```

Comment:
See comment to Patriks comment `crossPaths == false` in akka makes this necessary.